code/game/turfs/flooring/flooring.dm

```diff
@@ -275,6 +280,23 @@ var/list/flooring_types
 		'sound/effects/footstep/wood4.ogg',
 		'sound/effects/footstep/wood5.ogg'))
 
+/decl/flooring/sifwood
```

Comment:
Just asking; given all the flags made here, could this be a child of regular wood floor?

Reply:
I tried that, and it was having similar problems to the blue carpet, in that you couldn't put it down for some reason. It might not have anything to do with being a child of another type of turf but I made it its own separate thing in an attempt to wrap my head around why it wasn't working.